boxsdk/client/client.py

```diff
@@ -15,6 +14,7 @@
 from ..pagination.limit_offset_based_object_collection import LimitOffsetBasedObjectCollection
 from ..pagination.marker_based_object_collection import MarkerBasedObjectCollection
 from ..util.shared_link import get_shared_link_header
+from ..object.item import Item
```

Comment:
Can this be removed? Linter is not passing because this is imported, but not used.